### PR TITLE
09: Page Redesign 2

### PR DESCRIPTION
**Pull request by gabrielizalo**
# Summay

* Updated html and scss files to change page layout with CSS Grid.

# Trello Card

https://trello.com/c/atVH4WGv/12-redise%C3%B1o-de-la-p%C3%A1gina

# Screenshot

screenshot 2018-09-25_14-55-38-486

### CODE REVIEW

File: maquetado/scss/_tech.scss [hunk 1]
```diff
@@ -1,7 +1,11 @@
+.technologies {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: You can replace this for: `repeat(3, 1fr)`

File: maquetado/scss/_tech.scss [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
+  grid-template-rows: 1fr 1fr;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: Same here, replace it for: `repeat(2, 1fr)`

File: maquetado/scss/_tech.scss [hunk 3]
```diff
@@ -1,7 +1,11 @@
...
+}
+
 .technology {
   align-items: center;
-  background-color: $green;
-  border-radius: $tech-border-radius;
   display: flex;
   flex-direction: column;
   margin: 20px;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: Delete this property and in `.technologies` class add `grid-gap` property

File: maquetado/scss/_tech.scss [hunk 4]
```diff
@@ -10,21 +14,11 @@
   position: relative;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: Delete this property isn't necessary

File: maquetado/scss/_tech.scss [hunk 5]
```diff
@@ -10,21 +14,11 @@
...
-    top: -10px;
-    transition: transform 500ms ease;
-    width: 100%;
-    z-index: -1;
-  }
-
-  &:hover:before {
-    transform: translate(20px, 20px);
+  &:first-child {
+    grid-column: 1 / 1;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: This property isn't necessary. Delete it

File: maquetado/scss/_tech.scss [hunk 6]
```diff
@@ -10,21 +14,11 @@
...
+    grid-row: 1 / 3;
+    justify-content: center;
+    border-right: $green 2px solid;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: Change the definition of this property for: `2px solid $green`

**Review by henryzarza — CHANGES_REQUESTED** [review]
Good work 👍

**Review by mcallegari10 — APPROVED** [review]

**Review by henryzarza — APPROVED** [review]

**Pull request merged**